tofhir-server/src/main/scala/io/tofhir/server/endpoint/CodeSystemEndpoint.scala

```diff
@@ -68,13 +77,73 @@ class CodeSystemEndpoint(codeSystemRepository: ICodeSystemRepository) extends La
     }
   }
 
+  /**
+   * Route to update code system csv header
+   * @param terminologyId terminology id
+   * @param codeSystemId code system id
+   * @return
+   */
+  private def updateCodeSystemHeader(terminologyId: String, codeSystemId: String): Route = {
```

Comment:
Can you please follow the same convention while naming the functions inside an Endpoint class?